pkg/cache/dapr/dapr.go

```diff
@@ -82,6 +82,10 @@ func (d *daprCache) Set(ctx context.Context, key string, value interface{}) bool
 }
 
 func (d *daprCache) SetWithTTL(ctx context.Context, key string, value interface{}, ttl time.Duration) bool {
+	if ttl < 0 {
+		logger.GetLogger(ctx, logOpt).Errorf("Error saving value to redis: ttl provided is invalid: %v", ttl)
```

Comment:
should we update the msg to " TTL must be greater >= 0 since this is the condition we are looking for